app/views/job_runs/show.html.erb

```diff
@@ -24,11 +24,15 @@
     <% end %>
     <dt class="col-sm-2">Job Output Log</dt>
     <dd class="col-sm-10">
-    <% if @job_run.output_location %>
-      <%= link_to 'Download', download_path(@job_run) %>
+    <% if @job_run.complete? %>
+      <%= link_to 'Download', download_log_path(@job_run) %>
     <% else %>
       Job is not yet complete.  Please check back later.
     <% end %>
     </dd>
+    <% if @job_run.complete? && @job_run.job_type == 'discovery_report' %>
+      <dt class="col-sm-2">Discovery Report</dt>
+      <dd class="col-sm-10"><%= link_to 'Download', download_report_path(@job_run) %></dd>
+    <% end %>
   </dl>
```

Comment:
show the links when the job is complete (each job gets the progress log file, discovery report also gets the report file)